lib/nitcorn/examples/src/nitcorn_reverse_proxy.nit

```diff
@@ -0,0 +1,9 @@
+import nitcorn
+
+var vh = new VirtualHost("localhost:8080")
+var proxy_interface = new Interface("localhost", 31337)
+vh.routes.add new Route("/", new ProxyAction(proxy_interface))
+
```

Comment:
You'll need to add the following code snippet to pass the test, actually to avoid executing you server when running the tests:

~~~
# Avoid executing when running tests
if "NIT_TESTING".environ == "true" then exit 0
~~~